src/papi-client.ts

```diff
@@ -60,9 +64,9 @@ export class PapiClient {
         let callback = (err, res) => {
             if (err) {
                 deferred.reject(err);
-            } else if (res.body.status === "ERROR") {
+            } else if (res.body.status === BatchOperationStatus[BatchOperationStatus.ERROR]) {
```

Comment:
It seems a good idea but `BatchOperationStatus[res.body.status]` is resolved as a string by typescript.
`error TS2365: Operator '===' cannot be applied to types 'string' and 'BatchOperationStatus'`